core-tests/src/test/kotlin/net/corda/coretests/flows/FlowExternalOperationStartFlowTest.kt

```
@@ -42,6 +42,8 @@ class FlowExternalOperationStartFlowTest : AbstractFlowExternalOperationTest() {
 
         @Suspendable
         override fun testCode(): Any {
+            // shadow member variable to avoid fiber serialisation errors
```

What are the issues with this test? Needs fixing if something is wrong!!

---

If the member variable is used (as before), it now fails due to fiber serialisation with `ArrayIndexOutOfBoundsException: -1`. See the PR description for some more details.

Looking at how `serviceHub` is used, I initially thought that's kind of expected. But judging from your reaction, I guess it's not.. I'll reset this for now and let the tests fail and we can investigate and decide what we do later on. But, overall I think there's some deeper bug here that was just not manifesting before.

---

Uhhh ye this looks like the error from the jira you linked in the description. I'll take a look at it.